### ISSUE TITLE
[no task] dest for run doesn't work as expected when src path is an array

### ISSUE THREAD

**Issue by dracupid** [ysmood/nokit#4]
```coffee
task 'test', ->
        kit.warp ['doc/*', 'lib/*']
        .run 'dest'
```
Only `doc` is copied to dest, `lib` is written to the origin folder.
![2015-04-29 21 23 06 1](https://cloud.githubusercontent.com/assets/5706703/7391771/09cccfba-eeb6-11e4-8408-8aafffc61cbf.png)

**Comment by ysmood** [comment]
It's a bug of `nofs`, fixed 67504e468a2526d1368527860864cb05b3f843d7

**Comment by ysmood** [comment]
The `v0.6.1` released.
